insights/parsers/ssh.py

```diff
@@ -79,14 +79,22 @@ class SshDConfig(Parser):
 
     def parse_content(self, content):
         self.lines = []
+        content_iter = iter(content)
         for line in get_active_lines(content):
+            raw_line = next(
+                (r for r in content_iter if r.lstrip().startswith(line.split()[0])),
+                line
+            )
```

Comment:
The get_active_lines and the inner-loop `next(..for..)` comprehension, will iterate the whole content at least twice. It maybe in bad efficient especially when the `content` is large.

How about just like what you did in the first update, remove the `get_active_lines` and iterate the lines with custom loop?

e.g.,
```
 for line in content:
    raw_line = line.strip()
    if raw_line:
        line = raw_line.split(comment_char, 1)[0].strip()
        line_splits = [s.strip() for s in re.split(r"[\s=]+", line, 1)]
        ...


```